lochness/rpms/__init__.py

```diff
@@ -133,6 +133,16 @@ def get_run_sheets_for_datatypes(target_df_loc: Union[Path, str]) -> None:
             target_df['timepoint'] = target_df['visit'].map(time_to_timepoint)
             for tp, table in target_df.groupby('timepoint'):
                 run_sheet_output = f'{run_sheet_output_prefix}_{tp}.csv'
+
+                # compare existing table
+                if Path(run_sheet_output).is_file():
+                    run_sheet_prev = pd.read_csv(
+                        run_sheet_output, dtype=str).reset_index(drop=True)
+                    same_df = table.astype(str).equals(
+                            run_sheet_prev.astype(str))
```

Comment:
Is `dtype=str` and `astype(str)` for converting the whole data frame to string?

Reply:
Yes. I thought the `dtype=str` in `pd.read_csv() alone would solve the issue, but it required `.astype(str)` in order to completely match the datatype.

Reply:
> but it required

This is a learning for me.